docs/src/gui/gladevcp.adoc

```diff
@@ -1509,26 +1523,40 @@ Reference Type::
   * `distance-to-go` (relative to current user coordinate origin).
 Joint Number::
   Used to select which axis (technically which joint) is displayed. +
-  On a trivialkins machine (mill, lathe, router) axis vrs joint number are:
+  On a trivialkins machine (mill, lathe, router) axis vs. joint number are:
 +
   0:X  1:Y  2:Z  3:A  4:B  5:C  6:U  7:V  8:W
-Display units::
-  Used to toggle the display units between metric and imperial.
+Font Family::
+  Specify the font family e.g. mono. Defaults to sans.
+  If the font does not exist then the current system font will be used.
+Font Size::
+  Specify the size of the font between 8 and 96. Defaults to 40
+Font Weight::
+  Specify the weight of the font. Select from lighter, normal, bold, or bolder.
+  Defaults to bold
+Unhomed Color::
+  The text color when unhomed pecified as a Gdk.RGBA color. Defaults to red,
+  Gdk.RGBA(red=1.000000, green=0.000000, blue=0.000000, alpha=1.000000)
+Homed Color::
+  The text color when homed pecified as a Gdk.RGBA color.   Defaults to green,
+  Gdk.RGBA(red=0.000000, green=0.501961, blue=0.000000, alpha=1.000000)
```

Comment:
Wouldn't it be easier to use if we have here the "real" property names here like `font_family` and so on?
I know it's not really consequently used here, but e.g. in Combi_DRO.